plugin/grepper.vim

```diff
@@ -806,11 +806,11 @@ function! s:finish_up(flags)
   call s:restore_errorformat()
 
   try
-    let title = has('nvim') ? cmdline : {'title': cmdline}
+    let attrs = has('nvim') ? cmdline : {'title': cmdline, 'context': @/}
```

Comment:
This line could omit the `nvim` check, setting `attrs` unconditionally:
``` vim
let attrs = {'title': cmdline, 'context': @/}
```
At least on the version of neovim I tried (v0.2.2), the last argument is a dictionary, not a string, and unrecognized keys are simply ignored.

Reply:
Back in v0.1.7 it was still a string, and the following release, v0.2.0, is 1838 commits but not even a year ago. I usually take Debian stable as a reference and of course it still ships v0.1.7.. I'll change that in a follow-up commit. Thanks for the pointer!